src/main.ts

```diff
@@ -175,7 +179,7 @@ function setupSigns(viewport: Viewport) {
 			wordWrapWidth: 550,
 			align: 'center',
 		},
-		x: -280,
+		x: -290,
```

Comment:
Center it a bit better on firefox, because bold weight for this font is rendered differently for chrome and firefox lol